passkey-client/src/lib.rs

```diff
@@ -209,6 +212,9 @@ where
                 None
             };
 
+        let uv = request.authenticator_selection.map(|s| s.user_verification)
+            != Some(UserVerificationRequirement::Discouraged);
```

Comment:
**Suggestion(follow-up):** So this would put `uv = true` on `preferred` and `required`. This may not be true for other providers, they may want to do UV only on `required` for example. This should be a setting on the client that can be enabled. Not a blocking issue for this PR, I'm mainly adding this to create a follow-up issue about it.

Reply:
Upcoming PRs make changes to this logic. I've also added a comment about this in #25 so let's resolve this comment and continue the discussion there!